### PR TITLE
Change the defibrillator "return to body" message sound from adminhelp.ogg to bikehorn.ogg

### PR DESCRIPTION
**Pull request by Misha600**
## About The Pull Request

Change the defibrillator sound you get when you're a ghost and someone's trying to defib you from adminhelp.ogg to bikehorn.ogg

## Why It's Good For The Game

People might think that an admin is yelling at them and they can't see the message. Changing it to bikehorn will still get your attention without knocking years off of your lifespan from the absolute terror the sound causes

## Changelog
:cl:
qol: changed the defibrillator return to body message sound from the bwoink sound to the bikehorn sound
/:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Why my man. Admin OGG in fact gets the person in attention. I would rather want you to make the Enter or jump option and the entire defib message bigger in text size.

File: code/game/objects/items/defibrillator.dm [hunk 1]
```diff
@@ -193,7 +193,7 @@
 
 	var/mob/dead/observer/G = H.get_ghost()
 	if(istype(G))
-		notify_ghost(G, "<font size=3>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/effects/adminhelp.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
+		notify_ghost(G, "<font size=3>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/items/bikehorn.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
```
Comment (SpaceLoveSs13) [inline code/game/objects/items/defibrillator.dm:196]:
```suggestion
		notify_ghost(G, "<font size=8>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/items/bikehorn.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
```

Comment (SpaceLoveSs13) [reply code/game/objects/items/defibrillator.dm:196]: Do this maybe

Comment (Misha600) [reply code/game/objects/items/defibrillator.dm:196]: yeah that'll probably work better, thanks

Comment (Misha600) [reply code/game/objects/items/defibrillator.dm:196]: > Why my man. Admin OGG in fact gets the person in attention. I would rather want you to make the Enter or jump option and the entire defib message bigger in text size.

it'll get you in attention at the cost of making you think an admin's trying to yell at you until you realize someone's trying to revive you, it'll also make you think someone's trying to revive you if an admin's trying to yell at you

**Comment by Luxyboo** [comment]
as neat as this is i agree with space love, plus there are already things in game that make a bike horn noise, maybe a way to choose your own defib revive noise.

**Comment by Misha600** [comment]
> as neat as this is i agree with space love, plus there are already things in game that make a bike horn noise, maybe a way to choose your own defib revive noise.

I would do that but i know nothing about DM and bikehorn was the first sound i thought of. There's no reason for it to be adminhelp and it's unnecessarily confusing. Adminhelp should be reserved for getting ahelped.

**Review by BraveMole — CHANGES_REQUESTED** [review]
I agree that ahelp sound should be reserved to ahelp.

Try to find a new sound, maybe on https://freesound.org/ ?

**Comment by Misha600** [comment]
> I agree that ahelp sound should be reserved to ahelp.
> 
> Try to find a new sound, maybe on https://freesound.org/ ?

Yeah, I should have done that in the first place. I'll reopen this once I get time to do that.

**Pull request closed without merging**